### ISSUE TITLE
Cannot run typescript app - EADDRINUSE, JS app works fine

### ISSUE THREAD

**Issue by Tomino2112** [Unitech/pm2#2385]
If reporting a bug, please use the following template.

#### Expected behaviour

Typescript app runs

#### Actual behaviour

Typescript app doesnt run and reports address in use

#### Steps to reproduce

Run typescript app

#### Software versions used

```
OS         : Ubuntu 16
node.js    : 6.5.0
PM2        : 2.0.12
```

#### PM2 Log output

Use the command: `tail --lines 50 ~/.pm2/pm2.log`

```
2016-09-16 13:56:18: Starting execution sequence in -fork mode- for app name:app id:0
2016-09-16 13:56:18: App name:app id:0 online
2016-09-16 13:56:31: App [app] with id [0] and pid [12225], exited with code [1] via signal [SIGINT]
2016-09-16 13:56:31: Starting execution sequence in -fork mode- for app name:app id:0
2016-09-16 13:56:31: App name:app id:0 online
2016-09-16 13:56:43: App [app] with id [0] and pid [12273], exited with code [1] via signal [SIGINT]
2016-09-16 13:56:43: Starting execution sequence in -fork mode- for app name:app id:0
2016-09-16 13:56:43: App name:app id:0 online
2016-09-16 13:56:55: App [app] with id [0] and pid [12311], exited with code [1] via signal [SIGINT]
2016-09-16 13:56:55: Starting execution sequence in -fork mode- for app name:app id:0
2016-09-16 13:56:55: App name:app id:0 online
2016-09-16 13:56:57: Stopping app:app id:0
2016-09-16 13:56:57: App [app] with id [0] and pid [12359], exited with code [0] via signal [SIGINT]
```

Running
```
pm2 start app.ts
```

Gives error:
```
0|app      | Error: listen EADDRINUSE :::3500
0|app      |     at Object.exports._errnoException (util.js:1026:11)
0|app      |     at exports._exceptionWithHostPort (util.js:1049:20)
0|app      |     at Server._listen2 (net.js:1253:14)
0|app      |     at listen (net.js:1289:10)
0|app      |     at Server.listen (net.js:1385:5)
0|app      |     at EventEmitter.listen (/vagrant/src/streaming/node_modules/express/lib/application.js:617:24)
0|app      |     at Object.<anonymous> (/vagrant/src/streaming/app.ts:47:40)
0|app      |     at Module._compile (module.js:556:32)
0|app      |     at Object.Module._extensions..js (module.js:565:10)
0|app      |     at Module.load (module.js:473:32)
0|app      |     at tryModuleLoad (module.js:432:12)
0|app      |     at Function.Module._load (module.js:424:3)
0|app      |     at Module.require (module.js:483:17)
0|app      |     at require (internal/module.js:20:19)
0|app      |     at Object.<anonymous> (/vagrant/src/streaming/sockets/sockets.ts:1:1)
0|app      |     at Module._compile (module.js:556:32)
```

but running 
```
pm2 start app.js
```

works fine

**Comment by Unitech** [comment]
ts-node upgraded to latest: https://github.com/Unitech/pm2/commit/50c114c81851194785ae7e249cfef4a42117ef14

```bash
$ npm install Unitech/pm2#development -g
```

**Comment by Unitech** [comment]
landed in latest public release (2.0.15)

```bash
$ npm install pm2 -g
```
